Samples/Sample XML/StudentEnrollment.xml

```diff
@@ -17776,7 +17776,7 @@
 		<HomeroomIndicator>1</HomeroomIndicator>
 		<DualCredit>
 			<DualCreditIndicator>1</DualCreditIndicator>
-			<DualCreditType>uri://ed-fi.org/DualCreditTypeDescriptor#Advanced Course Eligible for College Credit</DualCreditType>
+			<DualCreditType>uri://ed-fi.org/DualCreditTypeDescriptor#Advanced Courses Eligible for College Credit</DualCreditType>
```

Comment:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Data-Standard/blob/v5.2.0/Descriptors/DualCreditTypeDescriptor.xml#L3 CodeValue , ShortDescription and Description are not matching text here  . CodeValue , ShortDescription and Description should have same text comparing to other descriptors